src/_includes/partials/header.njk

```diff
@@ -1,40 +1,41 @@
 {% set otherLang = "fr" if locale == "en" else "en" %}
-<header>
-    <div  lang="{{ otherLang }}">
+<header class="header">
+    <div class="header__container" lang="{{ otherLang }}">
+        <a href="{% if locale == "en" %}/en/ {% else %}/fr/ {% endif %}" class="header__logo">
```

Comment:
We will need the Government of Canada signature link to canada.ca

Reply:
Done!